include/tgfx/layers/Layer.h

```diff
@@ -234,6 +234,19 @@ class Layer {
    */
   void setLayerStyles(const std::vector<std::shared_ptr<LayerStyle>>& value);
 
+  /**
+   * Returns true if the source of layer styles apply filters and layer styles. The default value is
+   * true.
```

Comment:
Whether to exclude child effects in the layer style. If true, child layer styles and filters are not included in the layer content used to generate the layer style. The default value is false.